orienteer-pivottable/src/main/java/org/orienteer/pivottable/PivotTableModule.java

```diff
@@ -38,19 +39,20 @@ public void onUpdate(OrienteerWebApplication app, ODatabaseDocument db,
 		switch (oldVersion+1)
 		{
 			case 1:
-				onUpdateToFirstVesion(app, db);
+				onUpdateToFirstVersion(app, db);
 				break;
 			default:
 				break;
 		}
 		if(oldVersion+1<newVersion) onUpdate(app, db, oldVersion + 1, newVersion);
 	}
 
-	public void onUpdateToFirstVesion(OrienteerWebApplication app, ODatabaseDocument db)
+	public void onUpdateToFirstVersion(OrienteerWebApplication app, ODatabaseDocument db)
```

Comment:
As described below: it's already not a first version. Please think carefully how to code onInstall() and onUpdate().